backend/core/handler/master_book.go

```
@@ -0,0 +1,32 @@
+package handler
+
+import (
+	"context"
+
+	"github.com/sandonemaki/my_read_memo_Go_API/backend/pkg/oapi"
+)
+
+func (h Core) ListMasterBooks(ctx context.Context, _ oapi.ListMasterBooksRequestObject) (oapi.ListMasterBooksResponseObject, error) {
+	// TODO: 実装
```

golangci-lint (2.2.2)

core/handler/master_book.go:10: Line contains TODO/BUG/FIXME: "TODO: 実装"

(godox)